examples/empirical_frechet_mean_uncertainty_s3.py

```diff
@@ -0,0 +1,184 @@
+"""
+Uncertainty of the empirical Fréchet mean on the sphere
+
+The variance of the Fréchet mean FM_n of a sample of n IID
+random variables of variance Var is decreasing more slowly
+in a sphere than in a Euclidean space.
+This example computes the  modulation factor
+     alpha = Var( FM_n) / ( n * Var)
+for isotropic distributions on hyperspheres or radius 0<r<Pi
+in the sphere s3.
+"""
+
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geomstats.backend as gs
+# import geomstats.visualization as visualization
+
+from geomstats.geometry.hypersphere import Hypersphere
+
+# n = number of samples for computing the mean
+# NN = number of trials for the stochastic integral of expectation
+# Ntheta = number of points on the curve sig_est(theta)
+
+def my_random_uniform_sphere(n):
```

Comment:
my_random_uniform_sphere was used for a test on the computational time. I removed it since it does not appear to be faster than the one already implemented.

n change to dim everywhere